src/HotChocolate/ElasticSearch/src/Data.ElasticSearch.Driver/Filters/IAbstractElasticClient.cs

```diff
@@ -1,15 +0,0 @@
-using HotChocolate.Data.Filters;
-
-namespace HotChocolate.Data.ElasticSearch.Filters;
-
-/// <summary>
-/// Represents an abstract wrapper around either a elastic search
-/// or open search client
-/// </summary>
-public interface IAbstractElasticClient
```

Comment:
Yeah I guess it was meant to support both of them. I basicalliy moved that capability to the IElasticSearchExecutable., because I like the way the Mongo integration works. 

The class NextExecutable uses the NEST nuget to create a query based on the parsed filters etc. I hope the open search implementation will be like copy paste and replacing the namespaces